setup.py

```diff
@@ -1,8 +1,10 @@
+from coala_decorators import VERSION
 from distutils.core import setup
+
 setup(
   name = 'coala_decorators',
   packages = ['coala_decorators'],
-  version = '0.1.1',
+  version = VERSION,
```

Comment:
Um, I'm fairly sure that this shouldnt work.
You cant import a non-python file like `from <pkg name> import FILENAME`. (I hope Im not wrong here)
In the `coala` code, theres a small line in `coalib/__init__.py` which reads the VERSION file and saves it in the VERSION and __version__ variables.
Check https://github.com/coala-analyzer/coala/blob/master/coalib/__init__.py#L5